boorufetch.go

```diff
@@ -27,21 +27,22 @@ const (
 type Rating uint8
 
 const (
-	Safe Rating = iota
+	General Rating = iota
+	Sensitive
```

Comment:
Safe being replaced by General is fine, but Sensitive should be the last one in the list to preserve backwards compatibility.

Reply:
Moved it to the end of the list